lib/services/internalEngine/index.js

```diff
@@ -427,18 +454,18 @@ class InternalEngine {
 
 function handleError(error) {
   if (error instanceof es.errors.NoConnections) {
-    return Promise.reject(new ServiceUnavailableError('es service is not connected'));
+    return Bluebird.reject(new ServiceUnavailableError('es service is not connected'));
   }
 
   if (error instanceof KuzzleError) {
-    return Promise.reject(error);
+    return Bluebird.reject(error);
   }
 
   if (error.status === 404) {
-    return Promise.reject(new NotFoundError(`Not found: ${error.body._type}/${error.body._id}`));
+    return Bluebird.reject(new NotFoundError(`Not found: ${error.body._type}/${error.body._id}`));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 98 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)